src/foundry/foundry.js/applications/formApplications/userConfig.d.ts

```diff
@@ -0,0 +1,51 @@
+import { ConfiguredDocumentClass } from '../../../../types/helperTypes';
+import { UserData } from '../../../common/data/data.mjs';
+
+declare global {
+  /**
+   * The User Configuration application provides a form used to allow the current client to
+   * edit preferences and configurations about a User entity (typically their own).
+   * @typeParam P - the type of the options object
+   */
+  class UserConfig<P extends FormApplication.Options = FormApplication.Options> extends FormApplication<
+    P,
+    UserConfig.Data,
+    ConfiguredDocumentClass<typeof User>
+  > {
+    /**
+     * @param user    - The User entity being configured.
+     * @param options - Additional rendering options which modify the behavior of the form.
+     */
+    constructor(user: ConfiguredDocumentClass<typeof User>, options?: Partial<P>);
+
+    /** @override */
+    static get defaultOptions(): typeof FormApplication['defaultOptions'];
```

Comment:
Please document the default value, similar to how this is done for most `Application`s. Also it should reference `DocumentSheet` instead of `FormApplication`, probably.